### DISCUSSION TITLE
Broken documentation links in old issues transferred from retired dotnet/corefx repo - still accessible somewhere?

### DISCUSSION THREAD

**Discussion by jedibob5** [dotnet/runtime#61012] [Q&A] [answered]
On [this old issue](https://github.com/dotnet/runtime/issues/26969) which was transferred to this repository, there is a link to some documentation on a compatibility change in .NET Core: https://github.com/dotnet/corefx/wiki/ApiCompat#systemxmlschema

This link points to a wiki page on the retired dotnet/corefx repository, which no longer exists. Were these docs migrated anywhere? I can't find anything similar on this repo.

**Comment by PathogenDavid** [comment, marked as answer]
You can still download the wiki by cloning it directly via `https://github.com/dotnet/corefx.wiki.git`. Here's a GitHub gist of [ApiCompat.md#systemxmlschema](https://gist.github.com/PathogenDavid/bfa525c051446248c84f400a66c80010#systemxmlschema).

As far as I can find, this document was not migrated to anywhere in this repo. By and large I think this document has probably been supplanted by the use of the [platform compatibility analyzer](https://docs.microsoft.com/en-us/dotnet/standard/analyzers/platform-compat-analyzer) and documentation on specific affected APIs.

**Reply by danmoseley** [reply]
@Anipik
